launcher/minecraft/mod/ResourceFolderModel.cpp

```diff
@@ -278,7 +344,8 @@ void ResourceFolderModel::resolveResource(Resource* res)
 
     connect(
         task.get(), &Task::succeeded, this, [=] { onParseSucceeded(ticket, res->internal_id()); }, Qt::ConnectionType::QueuedConnection);
-    connect(task.get(), &Task::failed, this, [=] { onParseFailed(ticket, res->internal_id()); }, Qt::ConnectionType::QueuedConnection);
+    connect(
+        task.get(), &Task::failed, this, [=] { onParseFailed(ticket, res->internal_id()); }, Qt::ConnectionType::QueuedConnection);
     connect(
         task.get(), &Task::finished, this,
         [=] {
```

Comment:
I'd much prefer the use of explicit capture args over the use of `[=]` it prevents the accidental capture of shared resources and extension of their lifetimes.
since we are updating this part of the code we might as well fix that now.

Reply:
Out of scope

Reply:
Wait no I did not touch this line at all and the file is filled with this